src/main/java/nodebox/function/NetworkFunctions.java

```diff
@@ -1,8 +1,18 @@
 package nodebox.function;
 
+import com.sun.org.apache.xerces.internal.impl.dv.xs.BooleanDV;
+import com.sun.org.apache.xpath.internal.operations.Bool;
```

Comment:
These two imports use proprietary, internal code. I don't think you use them inside of the code. Remove?